docs/examples/sample/README.md

```diff
@@ -0,0 +1,23 @@
+# LeaderWorkerSet Features
+LeaderWorkerSet supports the following features.
+
```

Comment:
Add one more example before these.

"Deploy a LeaderWorkerSet with 3 groups and 4 workers in each group"

And add the example yaml as well.